### PR TITLE
Rework jade getting started

### PR DESCRIPTION
**Pull request by rdebrand**
First real instructions on how to install the JADE software.

### CODE REVIEW

File: cernopendata/modules/fixtures/data/docs/jade-getting-started/jade-getting-started.md [hunk 1]
```diff
@@ -1,21 +1,24 @@
...
-download the JADE virtual machine image open it with VirtualBox. See
-instructions [here](/docs/jade-virtual-machine).
+JADE software needs to be run inside a container, e.g. with Docker or Podman. The corresponding GitHub repository can be found [here](https://github.com/andriish/JADE).
 
-## <a name="data">"What kind of JADE data will I work on?"</a>
+## <a name="docker">"Installing the JADE Software with Docker"</a>
 
-The data samples you can download from this portal consists all events
-collected by the JADE.
+It is required to have Docker installed and the repository cloned, e.g. with `git clone https://github.com/andriish/JADE`.
```
[the remark this answers is not in the document]
Comment (rdebrand) [reply cernopendata/modules/fixtures/data/docs/jade-getting-started/jade-getting-started.md]: Changed the depth, added a "size note" and the command now only clones a branch that contains just the software.

File: cernopendata/modules/fixtures/data/docs/jade-getting-started/jade-getting-started.md [hunk 2]
```diff
@@ -1,21 +1,24 @@
...
 
-## <a name="vbox">"I have installed VirtualBox, downloaded JADE VM image and launched it. And now?"</a>
+1. Pull the image with `docker pull ghcr.io/andriish/fedora39x86_64i686_gnu:latest` 
+2. Run the container `docker run -dit --platform linux/amd64 --name jade_soft -v $"(pwd)":/home ghcr.io/andriish/fedora39x86_64i686_gnu` \
+   The `-v` option will mount the current directory (where it might be necessary to use `"$PWD"` instead of `$"(pwd)"` due to a special             character error) to the home directory inside the container. It is thus useful to set the working directory to the cloned repository to          conveniently run the software scripts and programs. \
+   Note that not specifying a platform will result in the container software trying to run the image on the hosts platform type, while the image    requires to be run on the linux/amd64 platform. The `-dit` (detached, interactive, tty) option is also necessary to keep the container           running in the background and be interactive, else it might close immediately.
+4. Attach with `docker attach jade_soft` and run `sh jadeinstall.sh --bits=32` \
```
[the remark this answers is not in the document]
Comment (rdebrand) [reply cernopendata/modules/fixtures/data/docs/jade-getting-started/jade-getting-started.md]: Added a `cd` for now.

[the remark this answers is not in the document]
Comment (rdebrand) [reply cernopendata/modules/fixtures/data/docs/jade-getting-started/jade-getting-started.md]: It is unexpected, for me it works fine when I just follow the instructions.

Is the `--bits=3` intentional? Maybe `--bits=32` will fix it.

File: cernopendata/modules/fixtures/data/docs/jade-getting-started/jade-getting-started.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (andriish) [reply cernopendata/modules/fixtures/data/docs/jade-getting-started/jade-getting-started.md]: `-b master`, i.e. just one branch could be helpful as well

File: cernopendata/modules/fixtures/data/docs/jade-getting-started/jade-getting-started.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (tiborsimko) [reply cernopendata/modules/fixtures/data/docs/jade-getting-started/jade-getting-started.md]: Right, I must have made a typo :facepalm: I get success with `--bits=32`, and also when running tests.

**Review by tiborsimko — APPROVED** [review]
Thanks, rebased against `master`, fixed instructions (`cd /home/` only), fixed "software" upper/lowercase in titles, and fixed newlines in the itemised list by inserting HTML markup.

**Pull request merged**